tests/integration/protostar_fixture.py

```diff
@@ -3,50 +3,47 @@
 from argparse import Namespace
 from logging import getLogger
 from pathlib import Path
-from typing import Any, Callable, Dict, List, Optional, Tuple, cast, Union
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union, cast
 
 from pytest_mock import MockerFixture
 from starknet_py.net import KeyPair
 from starknet_py.net.models import StarknetChainId
 from starknet_py.net.signer.stark_curve_signer import StarkCurveSigner
 
-from protostar.self.protostar_directory import ProtostarDirectory
 from protostar.cli.map_targets_to_file_paths import map_targets_to_file_paths
 from protostar.commands import (
     BuildCommand,
     DeclareCommand,
     FormatCommand,
     InitCommand,
+    InvokeCommand,
     MigrateCommand,
 )
-from protostar.testing import TestingSummary
 from protostar.commands.deploy_command import DeployCommand
-from protostar.commands.test import TestCommand
 from protostar.commands.init.project_creator.new_project_creator import (
     NewProjectCreator,
 )
-from protostar.compiler import ProjectCompiler
+from protostar.commands.test import TestCommand
+from protostar.compiler import ProjectCairoPathBuilder, ProjectCompiler
 from protostar.compiler.compiled_contract_reader import CompiledContractReader
 from protostar.formatter.formatter import Formatter
 from protostar.formatter.formatting_result import (
     FormattingResult,
     format_formatting_result,
 )
 from protostar.formatter.formatting_summary import FormattingSummary
-from protostar.migrator import Migrator, MigratorExecutionEnvironment
-from protostar.protostar_toml import (
-    ProtostarContractsSection,
-    ProtostarTOMLWriter,
-)
-from protostar.starknet_gateway import Fee, GatewayFacadeFactory
 from protostar.io.input_requester import InputRequester
-
-from protostar.compiler import ProjectCairoPathBuilder
-from protostar.protostar_toml.protostar_project_section import ProtostarProjectSection
-from protostar.protostar_toml.io.protostar_toml_reader import ProtostarTOMLReader
 from protostar.io.log_color_provider import LogColorProvider
+from protostar.migrator import Migrator, MigratorExecutionEnvironment
+from protostar.protostar_toml import ProtostarContractsSection, ProtostarTOMLWriter
+from protostar.protostar_toml.io.protostar_toml_reader import ProtostarTOMLReader
+from protostar.protostar_toml.protostar_project_section import ProtostarProjectSection
+from protostar.self.protostar_directory import ProtostarDirectory
+from protostar.starknet_gateway import Fee, GatewayFacadeFactory
+from protostar.testing import TestingSummary
 
 
+# pylint: disable=too-many-instance-attributes
```

Comment:
how about removing this lint? we're disabling it all over the place 😬

Reply:
I am not sure. I found 4 similar comments in the code base.

Reply:
yeah exactly, so why do we need it?

Reply:
if it breaks the rule, we ignore it. so maybe it's better to disable it overall

Reply:
This rule encourages rethinking design. Anyway, disabling this rule is out of scope of this PR.